mitodl/odl-video-service
# Remove login and registration

#### What are the relevant tickets?
Fixes #311 

#### What's this PR do?
Removes login and register views

#### How should this be manually tested?
Go to `/`. It should redirect you to `/collections/` which should require you to login via Django admin if necessary.
